blockchain-scrapers/docker-compose.yml

```diff
@@ -78,6 +78,21 @@ services:
       restart_policy:
         delay: 2s
         window: 20s
+  neo:
```

Comment:
so you have twice neo as a service ? not sure I get it. shouldn't this one called differently like neonode ?